sql/database.sql

```diff
@@ -7640,3 +7640,585 @@ CREATE TABLE `form_clinical_instructions` (
+('Front Office', 'Popup:lists', '', '', '', 400, NULL),
+('Front Office', 'Miscellaneous:misimg', '', '', '', 500, NULL),
+('Front Office', 'About|/interface/main/about_page.php', '0', '0', 'File:file0', 0, NULL),
+('Front Office', 'Preferences|/interface/super/edit_globals.php?mode=user', '', '', 'File:file0', 100, NULL),
+('Front Office', 'Payment|/interface/patient_file/front_payment.php', '', '', 'Financial:top', 0, NULL),
+('Front Office', 'Checkout|/interface/patient_file/pos_checkout.php', '', '', 'Financial:top', 100, NULL),
+('Front Office', 'Front Rec|/interface/reports/front_receipts_report.php', '', '', 'Financial:top', 200, NULL),
+('Front Office', 'Pat Ledger|/interface/reports/pat_ledger.php?form=0', '', '', 'Financial:top', 500, NULL),
+('Front Office', 'Messages|/interface/main/messages/messages.php?form_active=1', '', '', 'Miscellaneous:misimg', 0, NULL),
+('Front Office', 'Patient Education|/interface/reports/patient_edu_web_lookup.php', '', '', 'Miscellaneous:misimg', 100, NULL),
+('Front Office', 'Patients|/interface/main/finder/dynamic_finder.php', '', '', 'Patient/Client:patimg', 0, NULL),
+('Front Office', 'New/Search|/interface/new/new.php', '', '', 'Patient/Client:patimg', 100, NULL),
+('Front Office', 'New/Patient|/interface/new/new.php', '', '', 'Patient/Client:patimg', 200, NULL),
+('Front Office', 'Clinical|/interface/reports/clinical_reports.php', '', '', 'Patient/Client:patimg', 300, NULL),
+('Front Office', 'Issues|/interface/patient_file/problem_encounter.php', '', '', 'Popup:lists', 0, NULL),
+('Front Office', 'Import|/custom/import_xml.php', '', '', 'Popup:lists', 100, NULL),
+('Front Office', 'Export|/custom/export_xml.php', '', '', 'Popup:lists', 200, NULL),
+('Front Office', 'Appts|/interface/reports/appointments_report.php?patient=', '', '', 'Popup:lists', 300, NULL),
+('Front Office', 'Superbill|/interface/patient_file/printed_fee_sheet.php?fill=1', '', '', 'Popup:lists', 400, NULL),
+('Front Office', 'Payment|/interface/patient_file/front_payment.php', '', '', 'Popup:lists', 500, NULL),
+('Front Office', 'Checkout|/interface/patient_file/pos_checkout.php', '', '', 'Popup:lists', 600, NULL),
+('Front Office', 'Letter|/interface/patient_file/letter.php', '', '', 'Popup:lists', 700, NULL),
+('Front Office', 'Chart Label|/interface/patient_file/label.php', '', '', 'Popup:lists', 800, NULL),
+('Front Office', 'Barcode Label|/interface/patient_file/barcode_label.php', '', '', 'Popup:lists', 900, NULL),
+('Front Office', 'Address Label|/interface/patient_file/addr_label.php', '', '', 'Popup:lists', 1000, NULL),
+('Front Office', 'Addr Book|/interface/usergroup/addrbook_list.php', '', '', 'View:', 100, NULL),
+('Front Office', 'Calendar|/interface/main/main_info.php', '', '', 'View:', 200, NULL),
+('Front Office', 'Flow Board|/interface/patient_tracker/patient_tracker.php?skip_timeout_reset=1', '', '', 'View:', 300, NULL);
+
+
+ALTER TABLE users ADD COLUMN fullscreen_role text NOT NULL;
```

Comment:
Just look for the users table and add the fields there. so when the users table is created it has the fields.

Reply:
Done!